data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/kill_fires_events.cfg

```diff
@@ -0,0 +1,43 @@
+#####

```

Comment:
Please add a `# wmllint: no translatables` followed by a blank line at the top of the file.